src/Backend/Modules/Tags/Entity/Tag.php

```diff
@@ -0,0 +1,208 @@
+<?php
+
+namespace Backend\Modules\Tags\Entity;
+
+/*
+ * This file is part of Fork CMS.
+ *
+ * For the full copyright and license information, please view the license
+ * file that was distributed with this source code.
+ */
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * This is the Tag Entity
+ *
+ * @author Jeroen Desloovere <info@jeroendesloovere.be>
+ *
+ * @ORM\Entity
+ * @ORM\Table(name="tags")
+ */
+class Tag
+{
+    /**
+     * @var int
+     *
+     * @ORM\Id
+     * @ORM\Column(type="integer", length=11)
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $language;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $tag;
+
+    /**
+     * @var int
+     *
+     * @ORM\Column(type="integer", length=11)
+     */
+    private $number;
+
+    /**
+     * @var string
+     *
+     * @ORM\Column(type="string", length=255)
+     */
+    private $url;
+
+    /**
+     * @ORM\OneToMany(targetEntity="Backend\Modules\Tags\Entity\TagConnection", mappedBy="tag", cascade={"persist", "remove"})
+     */
+    private $connections;
+
+    /**
+     * Constructor
+     */
+    public function __construct()
+    {
+        $this->connections = new \Doctrine\Common\Collections\ArrayCollection();
+    }
+
+    /**
+     * Gets id
+     *
+     * @return integer
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Gets the value of language.
+     *
+     * @return string
+     */
+    public function getLanguage()
+    {
+        return $this->language;
+    }
+
+    /**
+     * Sets the value of language.
+     *
+     * @param string $language the language
+     * @return self
```

Comment:
In Symfony, the phpDoc mostly contains `@return Tag` instead of `@return self`. I believe this is even a phpdoc best practice.